src/Http/include/Http/HttpDownloadManager.h

```diff
@@ -7,14 +7,8 @@
 
 #include <QNetworkAccessManager>
 #include <QObject>
-#include <QPointer>
-#include <memory>
 
 #include "Http/DownloadManager.h"
-#include "OrbitBase/CanceledOr.h"
-#include "OrbitBase/Future.h"
-#include "OrbitBase/Result.h"
-#include "OrbitBase/StopToken.h"
```

Comment:
Please add these includes back into the file. This only builds currently because DownloadManager.h adds these includes. This is called transitive includes and we should not do that. Each file should include what it uses. Not more, not less. The IWYU check helps with that. Details from the sponge link:

```

/mnt/github/orbitprofiler/src/Http/include/Http/HttpDownloadManager.h should add these lines:
#include <QString>                 // for QString
#include <filesystem>              // for path
#include <string>                  // for string
#include "OrbitBase/CanceledOr.h"  // for CanceledOr
#include "OrbitBase/Future.h"      // for Future
#include "OrbitBase/NotFoundOr.h"  // for NotFoundOr
#include "OrbitBase/Result.h"      // for ErrorMessageOr
#include "OrbitBase/StopToken.h"   // for StopToken

```